src/Components/Components/src/Reflection/ComponentProperties.cs

```diff
@@ -215,19 +214,23 @@ private static void ThrowForCaptureUnmatchedValuesConflict(Type targetType, stri
             throw new InvalidOperationException(
                 $"The property '{parameterName}' on component type '{targetType.FullName}' cannot be set explicitly " +
                 $"when also used to capture unmatched values. Unmatched values:" + Environment.NewLine +
-                string.Join(Environment.NewLine, unmatched.Keys.OrderBy(k => k)));
```

Comment:
Is this change in behavior OK?

Reply:
It's somewhat iffy. This is an error condition, so having some keys be ordered doesn't provide any additional QOL improvements per se. But, I'll let @SteveSandersonMS have his say.